packages/controls/src/widget_string.ts

```diff
@@ -346,8 +352,8 @@ class TextView extends DescriptionView {
     handleKeypress(e) {
         e.stopPropagation();
         // The submit message is deprecated in widgets 7
-        if (e.keyCode == 13) { // Return key
             this.send({event: 'submit'});
+        if (e.keyCode === 13) { // Return key
```

Comment:
@vidartf is this change intentional?

Reply:
It was a lint. Is there any advantage to using `==`?

Reply:
ah, I did not notice the new `=`.

You inverted two lines and there is nothing after the `if` statement,

Reply:
Must have happened during a rebase 😕  Fixed.